selfdrive/ui/tests/create_test_translations.sh

```diff
@@ -12,7 +12,6 @@ UNFINISHED="<translation type=\"unfinished\"><\/translation>"
 TRANSLATED="<translation>$TEST_TEXT<\/translation>"
 
 mkdir -p $UI_DIR/translations
-rm -f $TEST_TS_FILE $TEST_QM_FILE
```

Comment:
why was this removed?

Reply:
this can actually be put back in since the issues with the missing `main_test_en` file was from running `pytest test_translations`

Though we could also leave it as is since `lupdate -recursive "$UI_DIR" -ts $TEST_TS_FILE` both generates a `.ts` file and updates `$TEST_TS_FILE` if it already exists.

Reply:
the idea is that you have a fresh state every time

Reply:
makes sense, i added it back in.